packages/connector-chrome/src/chrome-launcher.ts

```diff
@@ -132,6 +132,8 @@ export class CDPLauncher extends Launcher {
             /* istanbul ignore next */
             if (isCI) {
                 this.chromeFlags.push('--headless', '--disable-gpu');
+            } else if (process.env.DOCKER === 'true') { // eslint-disable-line no-process-env
+                this.chromeFlags.push('--headless', '--disable-gpu', '--no-sandbox');
```

Comment:
Do we need `--disable-gpu` if we are in a linux container?  In the [FAQ](https://developers.google.com/web/updates/2017/04/headless-chrome#faq) it looks like is only for Windows